### DISCUSSION TITLE
Update apps with better themes integration

### DISCUSSION THREAD

**Discussion by oherau** [OnionUI/Onion#1508] [Ideas]
Right now the theme switcher works very well.
The main UI is updating accordingly with all the custom backgrounds, icons and fonts.
The 'Console' and 'Recent' sections show a fresh new design according to  users settings.
But apps such as the 'Quick Guide', the 'Package Manager' or the 'Activity Tracker' are still using the default theme.

I don't know if this improvement must be handled from the OnionUI project or on each and every app project.

**Comment by Aemiii91** [comment]
Hi, we are already in the process of adding theming to Activity Tracker. I'm not sure if we're gonna do Package Manager (it simply has too many custom elements).

Furthermore, we are in process of writing an entire new UI engine which will allow us much more flexibility, but this will take some time to finish (we're aiming for introducing it in Onion V5).

**Reply by oherau** [reply]
Whoa that's a very good new !
Anyway OnionUI v4.3 is already such a wonderful OS.
Any improvement is always a cherry on the cake but always welcome !
I can't wait to see OnionUI v5 !

Keep up the good work !
